--- configure.ac.orig	2011-02-04 15:59:01.000000000 -0500
+++ configure.ac	2011-02-04 15:59:01.000000000 -0500
@@ -552,9 +552,7 @@
 dnl Check for usual libc functions
 AC_CHECK_FUNCS([ctime_r daemon fcntl fdopendir fstatvfs fork getenv getpwuid_r gettimeofday isatty lstat memalign openat posix_fadvise posix_madvise posix_memalign setenv setlocale stricmp strnicmp tdestroy uselocale])
 AC_REPLACE_FUNCS([asprintf atof atoll getcwd getdelim getpid gmtime_r lldiv localtime_r nrand48 rewind strcasecmp strcasestr strdup strlcpy strncasecmp strndup strnlen strsep strtof strtok_r strtoll swab vasprintf])
-AC_CHECK_FUNCS(fdatasync,,
-  [AC_DEFINE(fdatasync, fsync, [Alias fdatasync() to fsync() if missing.])
-])
+AC_DEFINE(fdatasync, fsync, [Alias fdatasync() to fsync() if missing.])
 
 dnl Check for non-standard system calls
 AC_CHECK_FUNCS([accept4 dup3 eventfd vmsplice])
