--- src/modules/modules.c.orig	2011-02-04 15:59:01.000000000 -0500
+++ src/modules/modules.c	2011-02-04 15:59:01.000000000 -0500
@@ -824,6 +824,9 @@
     if( asprintf( &path, "%s" DIR_SEP "plugins", vlcpath ) != -1 )
         vlc_array_append( arraypaths, path );
 
+    if( asprintf( &path, "%s" DIR_SEP "plugins", PKGLIBDIR ) != -1 )
+        vlc_array_append( arraypaths, path );
+
     /* If the user provided a plugin path, we add it to the list */
     char *userpaths = var_InheritString( p_this, "plugin-path" );
     char *paths_iter;
